src/super_gradients/recipes/training_hyperparams/cifar10_resnet_train_params.yaml

```diff
@@ -2,7 +2,13 @@ defaults:
   - default_train_params
 
 max_epochs: 250
-lr_updates: [100, 150, 200]
+
```

Comment:
Don't think this file was supposed to be in this PR...

Reply:
This was part of previous PR that is now merged. Now it is removed fro this PR.